kgforge/core/archetypes/store.py

```diff
@@ -122,7 +122,7 @@ def __init__(
         self.token: Optional[str] = token
         self.versioned_id_template: Optional[str] = versioned_id_template
         loaded = (
-            self.mapping.load(file_resource_mapping) if file_resource_mapping else None
+            self.mapping.load_file(file_resource_mapping) if file_resource_mapping else None
```

Comment:
here, are you sure it is a file? it could be an url ...

Reply:
in fact I am getting the `FileNotFoundError`, for the mapping given in the configuration: `https://raw.githubusercontent.com/BlueBrain/nexus-forge/master/examples/configurations/nexus-store/file-to-resource-mapping.hjson`

Reply:
changing it back to `load` makes things work fine

Reply:
My mistake